conf/messages

```diff
@@ -209,9 +209,10 @@ goodsVatRate.Twenty = 20% VAT
 goodsVatRate.error.required = Select which VAT rate applies to the goods
 
 # SearchGoodsCountry
-searchGoodsCountry.title = In what country did you buy the {0}?
-searchGoodsCountry.heading = In what country did you buy the {0}?
-searchGoodsCountry.hint = If you bought the goods on a plane or boat, enter the country you were travelling from at the time of purchase.
+searchGoodsCountry.Export.title = What country are the goods going to? {0}?
+searchGoodsCountry.Export.heading = In what country did you buy the {0}?
```

Comment:
these are the wrong way round Import.title and Import.heading need to match and so to Export.title and Export.heading